content/meetings/2024-03-06/_index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: 6 March 2024
+geekdocAnchor: false
+weight: -8
+---
+<!--
+SPDX-FileCopyrightText: 2024 PNED G.I.E.
+
+SPDX-License-Identifier: CC-BY-4.0
+-->
+
+## Agenda
+- How to store URI and labels in CKAN?
+
+## Agreements
+- A new table will be created, and it will contain the URI, label, language code, and grouping_property.
```

Comment:
as agreed I have checked multilingual extension: it works for CKAN frontend but not for package API response. So probably we can reuse vocabulary table at least for now.